core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -490,4 +490,27 @@ public OrderController(UBSClientService ubsClientService) {
         return ResponseEntity.status(HttpStatus.OK)
             .body(ubsClientService.getCourierLocationByCourierIdAndLanguageCode(courierId));
     }
+
+    /**
+     * Controller checks if received data Client is valid and stores payment info if
+     * is.
+     *
+     * @param dto {@link PaymentResponseDto} - response order data.
+     * @return {@link HttpStatus} - http status.
+     * @author Max Boiarchuk
+     */
+    @ApiOperation(value = "Receive payment from  Client Fondy.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST)
+    })
+    @PostMapping("/receivePaymentClient")
+    public ResponseEntity<HttpStatus> receivePaymentClient(
+        PaymentResponseDto dto, HttpServletResponse response) throws IOException {
+        ubsClientService.validatePaymentClient(dto);
+        if (HttpStatus.OK.is2xxSuccessful()) {
+            response.sendRedirect("https://ita-social-projects.github.io/GreenCityClient/#/ubs/confirm");
```

Comment:
https://ita-social-projects.github.io/GreenCityClient/#/ubs/confirm - should be in properties.
https://ita-social-projects.github.io/GreenCityClient - should be replaced with some ${base_url} parameter.